### DISCUSSION TITLE
non-JSX style syntax for conditionals and loops in .astro template

### DISCUSSION THREAD

**Discussion by otonielguajardo** [withastro/roadmap#571] [Proposal]
# Summary

A way to write loops and conditionals on astro templates that doesn't use JSX like syntax would attract more developers and web designers to use Astro

# Background & Motivation

Some developers like a more explicit separation of languages when writing logic on templates, which JSX has not,
Also is easier to work with designers who already know css and html but aren't javascript developers

# Example

**Laravel style** 
```
@if($user->status =='waiting')         
      <p>Hold on!</p>         
@else
      <p>{{ $user->status }}</p>        
@endif
```

**Rails style** 
```
<ul>
  <% @books.each do |book| %>
        <li><%= book.title %></li>
  <% end %> 
</ul>
```

**Svelte style**
```
{#if x > 10}
	<p>{x} is greater than 10</p>
{:else}
	<p>{x} is less than 10</p>
{/if}
```

**Comment by ghost** [comment]
so https://github.com/withastro/roadmap/discussions/737?

**Comment by PierBover** [comment]
Anything would be better than the JSX syntax ugh
